scripts/upgrade

```diff
@@ -207,7 +207,7 @@ then
         mv "$tmpdir" "$install_dir"
 
         # Set write access for the following commands
-        chown -R $app: "$install_dir" "$data_dir"
+        chown -R $app:www-data "$install_dir"
```

Comment:
```suggestion
        chown -R $app:www-data "$install_dir"
        chmod og-rwx "$install_dir/config"
```

(me being paranoid about compromission of www-data that would result in leaking secrets etc)